lib/suites.js

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2019-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import {Ed25519Signature2018} from '@digitalbazaar/ed25519-signature-2018';
+import {Ed25519Signature2020} from '@digitalbazaar/ed25519-signature-2020';
+import {
+  cryptosuite as eddsa2022CryptoSuite
+} from '@digitalbazaar/eddsa-2022-cryptosuite';
+
+const createEddsa2022Suite = async ({signer}) => {
+  // remove milliseconds precision
+  const date = new Date().toISOString().replace(/\.\d+Z$/, 'Z');
+  const cryptosuite = eddsa2022CryptoSuite;
+  return new DataIntegrityProof({signer, date, cryptosuite});
+};
```

Comment:
This should be moved to the bottom of the file and use this syntax:

```js
async function _createEddsa2022Suite({signer}) {
}
```

Reply:
Agreed and address here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/97/commits/db785d4c9069d88cd817c383387762798033d2ba